File: SistemasProgramacionRFCVB/SistemasProgramacionRFCVB/Formularios/Contacto.vb
Public Class Contacto
    Inherits Persona
    Public telefono As Long
    Public correo As String

    Public Property TelefonoP() As Long
        Get
            Return telefono
        End Get
        Set(value As Long)
            telefono = value
        End Set
    End Property

    Public Property CorreoP() As String
        Get
            Return correo
        End Get
        Set(value As String)
            correo = value
        End Set
    End Property

    'Método constructor
    Public Sub Contacto(ByVal nombrePersona As String, ByVal apPaternoPersona As String, ByVal apMaternoPersona As String, ByVal fechaNacimientoPersona As DateTime, ByVal telefonoPersona As Long, ByVal correoPersona As String)
        NombreP = nombrePersona
        ApellidoPaternoP = apPaternoPersona
        ApellidoMaternoP = apMaternoPersona
        FechaNacimientoP = fechaNacimientoPersona
        TelefonoP = telefonoPersona
        CorreoP = correoPersona
    End Sub
End Class

File: SistemasProgramacionRFC/SistemasProgramacionRFC/Clases/Contacto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SistemasProgramacionRFC.Clases
{
    class Contacto : Persona
    {
        private long telefono;
        private string correo;

        public long Telefono
        {
            get { return telefono; }
            set { telefono = value; }
        }
        
        public string Correo
        {
            get { return correo; }
            set { correo = value; }
        }

        //Método constructor
        public Contacto(string nombrePersona,
                        string apPaternoPersona,
                        string apMaternoPersona,
                        DateTime fechaNacimientoPersona,
                        long telefonoPersona,
                        string correoPersona)
            : base(nombrePersona, apPaternoPersona, apMaternoPersona, fechaNacimientoPersona)
        {
            this.Telefono = telefonoPersona;
            this.Correo = correoPersona;
        }
    }
}
